agent/util-scripts/pbench-tool-meister-start

```diff
@@ -136,6 +136,10 @@ from pbench.agent.utils import (
 from pbench.common.utils import validate_hostname
 
 
+# The --orchestrate parameter default choice, and the full list of choices.
+_def_orchestrate_choice = "create"
+_orchestrate_choices = [_def_orchestrate_choice, "existing"]
```

Comment:
This doesn't necessarily need to gate the merge, but....

The value of `_orchestrate_choices` should be "complete" and not reference the default.  Instead, the default should select one of the values from the list (and there should be an assertion somewhere which ensures that the value is in the list).  The problem with the approach taken here is that the default cannot be changed without _also_ changing the list of choices (and naive attempts to do so will damage the list, by dropping a value and presumably duplicating another).